src/main/java/com/fasterxml/jackson/databind/PropertyName.java

```diff
@@ -242,10 +244,7 @@ public boolean equals(Object o)
 
     @Override
     public int hashCode() {
-        if (_namespace == null) {
-            return _simpleName.hashCode();
-        }
-        return _namespace.hashCode() ^  _simpleName.hashCode();
+        return Objects.hash(_namespace, _simpleName);
```

Comment:
It was replaced with a generic implementation.

Reply:
Ah, to avoid null check to simplify code. Ok, that's reasonable.